api/v1beta1/nutanixmachine_types.go

```diff
@@ -70,8 +93,13 @@ type NutanixMachineSpec struct {
 	// image is to identify the rhcos image uploaded to the Prism Central (PC)
```

Comment:
`rhcos` in this context meaning `Red Hat Enterprise Linux CoreOS`?
```suggestion
	// image is to identify the machine image uploaded to the Prism Central (PC)
```

Reply:
I copied and pasted what we previously used. I can change this.